.env.example

```diff
@@ -1 +1,7 @@
-PORT=
+DB_USERNAME=postgres
+DB_PASSWORD=db_password
+DB_NAME=barefoot_nomad
+DB_HOST=127.0.0.1
+PORT=5432
```

Comment:
Good work. Please remove the `PORT` value. The `PORT` env is meant for the app server, not the `DB` server.

Reply:
@rovilay Thanks, I've fixed it